Makefile

```diff
@@ -958,17 +958,39 @@ endif
 .PHONY: all
 all: distrib test_internal
 
+# Depending on which linker we're using,
+# we need a different invocation to get the
+# linker map file.
+ifeq ($(UNAME), Darwin)
+    MAP_FLAGS= -Wl,-map -Wl,$(BUILD_DIR)/llvm_objects/list.all
+    LIST_OUTPUT = > /dev/null
+else
+ifeq ($(OS), Windows_NT)
+    # This is for MinGW: the map file gets written, but the
+    # compilation fails with a file truncation error.  Instead,
+    # we use the old strategy.
+    # Note: we do in fact have to pass in this flag twice.
+    MAP_FLAGS= -Wl,-t -Wl,-t
+    LIST_OUTPUT = 2>&1 | grep "libLLVM" | grep ")" > $(BUILD_DIR)/llvm_objects/list.all
```

Comment:
Why are we grepping libLLVM twice in the mingw case?

Reply:
If we don’t grep initially in MinGW, the file size grows too large and causes an error, so it’s necessary during the compile command (the same error is the reason we can’t use the linker map). 

For the others, we filter post-compile. I figured it would be ok to redundantly grep for MinGW rather than introducing another difference between platforms.

Reply:
Might be worth recording that so that someone doesn't "fix" it later.

Reply:
Will do.